File: 100-Days-Series/Day 89/CountEvenOdd.cpp
#include<iostream>
#include<vector>
using namespace std;
// Functiont for count Odd & Even.
void getCountEvenOdd(vector<int>&arr){
    int size=arr.size();
    int evenCount=0;
    int oddCount=0;
    for(int i=0;i<size;i++){
        if(arr.at(i)%2==0){
            evenCount++;
        }else{
            oddCount++;
        }
    }
    cout<<"\nEven Numbers : "<<evenCount<<endl;
    cout<<"\nOdd Numbers : "<<oddCount<<endl;
}
// Let's Start Here
int main(){
    int sz;
    cout<<"\nEnter the size of the array : ";
    cin>>sz;
    vector<int>arr;
    cout<<"\nEnter the elements of the array :\n";
    for(int i=1;i<=sz;i++){
        cout<<"\nEnter "<<(i)<<" element : ";
        int val;
        cin>>val;
        arr.push_back(val);
    }
    getCountEvenOdd(arr);
}

File: 100-Days-Series/Day 89/CountEvenOdd.py

# Function for count Even & Odd
def get_count_even_odd(arr):
    countEven=0
    countOdd=0
    for i in arr:
        if i%2==0:
            countEven+=1
        else:
            countOdd+=1

    print("Even Numbers : ",countEven)
    print("Odd Numbers : ",countOdd)

# Let's Start Here
sz=int(input('Enter the size of array : '))
arr=[]
print('Enter elements of the array:')
for i in range(0,sz,1):
    print('Enter',(i+1),' element : ',end="")
    val=int(input())
    arr.append(val)
get_count_even_odd(arr)
